docker-storage-setup.sh

```diff
@@ -28,22 +28,22 @@ set -e
 #       expects the devices to be unpartitioned drives.  If "VG" is not
 #       specified, then use of the root disk's extra space is implied.
 #
-# VG:   The volume group to use for docker storage.  Defaults to the volume
-#       group where the root filesystem resides.  If VG is specified and the
-#       volume group does not exist, it will be created (which requires that
-#       "DEVS" be nonempty, since we don't currently support putting a second
-#       partition on the root disk).
+# VG:   The volume group to use for container runtime storage.
+#       Defaults to the volume group where the root filesystem resides.  If VG
+#       is specified and the volume group does not exist, it will be created
+#       (which requires that "DEVS" be nonempty, since we don't currently
+#       support putting a second partition on the root disk).
 #
 # The options below should be specified as values acceptable to 'lvextend -L':
 #
 # ROOT_SIZE: The size to which the root filesystem should be grown.
 #
-# DATA_SIZE: The desired size for the docker data LV.  Defaults to using all
-#            free space in the VG after the root LV and docker metadata LV
-#            have been allocated/grown.
+# DATA_SIZE: The desired size for the container runtime data LV.
+#            Defaults to using all free space in the VG after the root LV and
+#            container runtime metadata LV have been allocated/grown.
```

Comment:
I think this information needs to be updated. First of all now it is effectively size of thin pool. Also it defaults to using 40% of FREE space and not 100% of FREE space.